subsys/nrf_security/src/drivers/nrf_cc3xx/Kconfig

```diff
@@ -320,6 +338,12 @@ config PSA_NEED_CC3XX_RSA_PKCS1V15_CRYPT
 	depends on PSA_WANT_ALG_RSA_PKCS1V15_CRYPT
 	depends on PSA_USE_CC3XX_ASYMMETRIC_DRIVER
 
+config PSA_NEED_CC3XX_RSA_DRIVER
```

Comment:
If you want to keep this please add a comment to say that it was only created for some legacy integration. Otherwise you can just move it's depends on the legacy Kconfig file. I am ok with both as long as it has a comment to explain it in case that we keep it.